external/dhcpc/dhcpc.c

```diff
@@ -470,6 +482,9 @@ static void dhcpc_close(void *handle)
 
 static struct dhcpc_state *g_pResult;
 int g_dhcpc_state;
+int sock;
+int ioctl_ret;
```

Comment:
if it's global variable then add 'g_' prefix. and define global variables top of functions.
and why did you set sock to global variable?

Reply:
corrected